package.json

```diff
@@ -12,6 +12,7 @@
     "core-js": "^3.6.5",
     "vue": "^2.6.11",
     "vue-router": "^3.2.0",
+    "vue2-google-maps": "^0.10.7",
```

Comment:
I'm glad we went with this NPM package! The documentation was pretty thorough and implementation wasn't as tough as we thought it would be.  Not bad!